editoast/src/chartos/layers_description.rs

```diff
@@ -0,0 +1,49 @@
+use serde::{Deserialize, Serialize};
+use serde_yaml::{self};
+use std::fs;
+
+// select C.stuff from A inner join B C on C.id = C.id;
+//                       \___________________________/
+//                             a join expression
+//                            C is an alias for B
+type JoinExpr = String;
+
+pub const LAYERS_DESCRIPTION_PATH: &str = "./layers_description.yml";
+
+fn empty_vec() -> Vec<String> {
+    vec![]
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
+pub struct View {
+    pub name: String,
+    pub on_field: String,
+    pub data_expr: String,
+    #[serde(default = "empty_vec")]
```

Comment:
Few things here:
- `Vec` struct implements the trait `Default` which allow you to do something like `Vec::default()`.
- The default implementation of a Vec returns empty vec.

For these two reasons you don't need to specify a default function.

```suggestion
    #[serde(default)]
```